src/components/views/rooms/NewRoomIntro.tsx

```diff
@@ -166,7 +177,29 @@ const NewRoomIntro = () => {
         </React.Fragment>;
     }
 
+    function openRoomSettings(event) {
+        event.preventDefault();
+        dis.dispatch({
+            action: "open_room_settings",
+            initial_tab_id: ROOM_SECURITY_TAB,
+        });
+    }
+
+    const sub2 = _t("Your private messages are normally encrypted, but this room isn't. "+
+    "Usually this is due to an unsupported device or method being used, " +
+    "like email invites. <a>Enable encryption in settings.</a>", {},
+    { a: sub => <a onClick={openRoomSettings} href="#">{sub}</a> });
```

Comment:
My brain expects some indentation for a multi-line statement like this... perhaps something like:

```suggestion
    const sub2 = _t(
        "Your private messages are normally encrypted, but this room isn't. "+
        "Usually this is due to an unsupported device or method being used, " +
        "like email invites. <a>Enable encryption in settings.</a>", {},
        { a: sub => <a onClick={openRoomSettings} href="#">{sub}</a> },
    );
```